src/common/ImageAvatar.js

```diff
@@ -31,14 +44,17 @@ export default class ImageAvatar extends PureComponent<Props> {
     const borderRadius =
       shape === 'rounded' ? size / 8 : shape === 'circle' ? size / 2 : shape === 'square' ? 0 : 0;
 
+    const imageSource = isLoading ? userDefaultAvatar : { uri: avatarUrl };
+
     return (
       <View>
         <Touchable onPress={onPress} style={touchableStyle}>
           <ImageBackground
             style={touchableStyle}
-            source={{ uri: avatarUrl }}
+            source={imageSource}
             resizeMode="cover"
             borderRadius={borderRadius}
+            onLoadEnd={(e) => { this.setState({ isLoading: false }); }}
```

Comment:
If loads fails, then it's better to show defaultAvatar.
So `onLoad` will be more appropriate here.
https://facebook.github.io/react-native/docs/image.html#onload

Reply:
I have changed it to `onLoad`